moztelemetry/dataset.py

```diff
@@ -70,6 +71,9 @@ class Dataset:
         schema = ['submissionDate', 'docType', 'platform']
 
         records = Dataset(bucket, schema) \\
+            .select('clientId',
+                    os_name='"environment.system".os.name',
```

Comment:
I am not sure why this is needed since we "stitch" the message back together in [here](https://github.com/mozilla/python_moztelemetry/blob/master/moztelemetry/heka/message_parser.py#L27). Ideally this should work without exposing the ugly raw structure of the message as our Python users are not aware of it.